Cython/Coverage.py

```diff
@@ -54,6 +54,11 @@
 
 from coverage.plugin import CoveragePlugin, FileTracer, FileReporter  # requires coverage.py 4.0+
 from coverage.files import canonical_filename
+try:
+    import coverage.tracer  # we mainly do this so that runtests can identify if coverage won't work
+except ImportError:
+    raise ImportError("Installed 'coverage' does not support plugins. "
+                      "See https://coverage.readthedocs.io/en/7.6.11/install.html#c-extension")
```

Comment:
This works as well:
```suggestion
                      "See https://coverage.readthedocs.io/en/latest/install.html#c-extension")
```
I'm not sure which is better. The `latest` link is guaranteed to provide up-to-date documentation but risks breaking at some point. The concrete version link will probably stay unchanged but might get outdated over time.
I'm slightly leaning towards the `latest` link to at least get current documentation, whatever it will be at that point.